CONTRIBUTING.md

```diff
@@ -179,7 +180,16 @@ the hook as follows from the main `cve-bin-tool` directory:
 pre-commit install
 ```
 
-Once this is installed, all of those commands will run automatically when you run `git commit` and it won't let you commit until any issues are resolved.  (You can also run them manually using `pre-commit` with no arguments.) This will only run on files staged for commit (e.g. things where you've already run `git add`).  If you want to run on arbitrary files, see below:
+Once this is installed, all of those commands will run automatically when you run `git commit` and it won't let you commit until any issues are resolved.  (You can also run them manually using `pre-commit` with no arguments.) This will only run on files staged for commit (e.g. things where you've already run `git add`).
+
+To run conventional-pre-commit to check commit messages, you can install the hook as follows from the main `cve-bin-tool` directory:
```

Comment:
I'm not sure we need to specify how to install just this hook specifically, `pre-commit install` from above should be enough to install all hooks.

This might be a bit more confusing for newcomers as well, after reading this at first I thought that to install this hook one needs to run the command below in addition to the one above.

Reply:
Seems like I'm wrong again. I tested the last time by running `pre-commit uninstall` and then `pre-commit install`. This only removes the script from `.git/hooks/pre-commit` but not from `.git/hooks/commit-msg` where the script required for `conventional-pre-commit` exists, so it seemed like `pre-commit install` installs all hooks but in reality it doesn't.

So the better option would be `pre-commit install` with something like `pre-commit install --hook-type commit-msg`. Refer https://github.com/pre-commit/pre-commit/issues/1139#issuecomment-534699310

Reply:
Oh, that's not very intuitive, yeah.

Combining both together is the way I prefer too 👍